lib/render/RayCaster.cpp

```diff
@@ -940,15 +1181,44 @@ void RayCaster::_renderTriangleStrips() const
             indexBuffer[ idx++ ] = y * bx + x;
             indexBuffer[ idx++ ] = (y + 1) * bx + x;
         }
+        if( attrib1 )
+        {
+            for( unsigned int x = 0; x < bx; x++ )
+            {
+                unsigned int   attribIdx       = (y * bx + x) * 4;
+                attrib1Buffer[ attribIdx     ] = int(x) - 1;
+                attrib1Buffer[ attribIdx + 1 ] = int(y);
+                attrib1Buffer[ attribIdx + 2 ] = 0;
+                attrib1Buffer[ attribIdx + 3 ] = 1;
+                               attribIdx       = ((y + 1) * bx + x) * 4;
+                attrib1Buffer[ attribIdx     ] = int(x) - 1;
+                attrib1Buffer[ attribIdx + 1 ] = int(y);
+                attrib1Buffer[ attribIdx + 2 ] = 0;
+                attrib1Buffer[ attribIdx + 3 ] = 1;
+            }
+            glBufferData( GL_ARRAY_BUFFER,      bx * by * 4 * sizeof(int),
+                          attrib1Buffer,        GL_STREAM_READ );
+            glVertexAttribIPointer( 1, 4,       GL_INT, 0, (void*)0 );
+        }
         glBufferData( GL_ELEMENT_ARRAY_BUFFER,  numOfVertices * sizeof(unsigned int),
-                      indexBuffer,              GL_STREAM_DRAW );
+                      indexBuffer,              GL_STREAM_READ );
```

Comment:
GL_STREAM_READ specifies you are writing data to the buffer from the GPU and reading it back to the CPU. You want to use GL_STREAM_DRAW

Reply:
addressed by commit 03bed17f.